Update many callouts to use <Tip>, <Note>, and <EnterpriseFeature> (#680)
apollographql/docs: src/content/graphos/basics/quickstart/next-steps.mdx

After you [create a supergraph](./cloud/) with GraphOS Studio, read below to learn about some of the next steps you should take to get the most out of it and the entire GraphOS platform.

<Note>

> For the most part, you can perform these next steps in any order. But you should definitely start by [installing the Rover CLI](#first-install-the-rover-cli).

Reviewer: ```suggestion
You should always start by  [installing the Rover CLI](#first-install-the-rover-cli). Otherwise, you can perform the steps described on this page in any order.
```
